internal/libhive/run.go

```diff
@@ -199,13 +199,17 @@ func (r *Runner) run(ctx context.Context, sim string, env SimEnv) (SimResult, er
 	defer shutdownServer(server)
 
 	// Create the simulator container.
+	envVars := map[string]string{
+		"HIVE_SIMULATOR":    "http://" + server.Addr().String(),
+		"HIVE_PARALLELISM":  strconv.Itoa(env.SimParallelism),
+		"HIVE_LOGLEVEL":     strconv.Itoa(env.SimLogLevel),
+		"HIVE_TEST_PATTERN": env.SimTestPattern,
+	}
+	if env.SimRandomSeed > 0 {
+		envVars["HIVE_RANDOM_SEED"] = strconv.Itoa(env.SimRandomSeed)
```

Comment:
We should just always define this in envVars, even if zero.

Reply:
Addressed in last commit. Simulators should now handle the random seed value being set to zero and behave accordingly if necessary.

Reply:
But does it matter that it's zero? Don't we just always want deterministic randomness?

Reply:
Yes I agree, it should always start with the same value. I think we can even improve this a bit by appending the Rand object to the Testing object.